src/xtgeo/well/wells.py

```diff
@@ -25,8 +25,6 @@ class Wells(object):
     def __init__(self, *args, **kwargs):
 
         self._wells = []  # list of Well objects
-        self._bw = False
```

Comment:
Are there no potential side effects from removing these?

Reply:
Potential, but naming is private and I haven't found any code that uses these.

Reply:
I will revert this change just in case.